src/mono/wasm/Wasm.Build.Tests/IcuTests.cs

```diff
@@ -232,15 +232,19 @@ public void FullIcuFromRuntimePackWithInvariant(BuildArgs buildArgs, bool invari
     }
 
     [Theory]
-    [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { false, RunHost.NodeJS | RunHost.Chrome })]
+    [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { false, RunHost.NodeJS | RunHost.Chrome })] 
     [MemberData(nameof(FullIcuWithICustomIcuTestData), parameters: new object[] { true, RunHost.NodeJS | RunHost.Chrome })]
     public void FullIcuFromRuntimePackWithCustomIcu(BuildArgs buildArgs, bool fullIcu, RunHost host, string id)
     {
         string projectName = $"fullIcuCustom_{fullIcu}_{buildArgs.Config}_{buildArgs.AOT}";
         bool dotnetWasmFromRuntimePack = !(buildArgs.AOT || buildArgs.Config == "Release");
 
         buildArgs = buildArgs with { ProjectName = projectName };
-        buildArgs = ExpandBuildArgs(buildArgs, extraProperties: $"<WasmIcuDataFileName>{s_customIcuPath}</WasmIcuDataFileName><WasmIncludeFullIcuData>{fullIcu}</WasmIncludeFullIcuData>");
+        string customFile = fullIcu ? "icudt_nonexisting.dat" : s_customIcuPath;
+        buildArgs = ExpandBuildArgs(
+            buildArgs, 
+            extraProperties: 
+                $"<WasmIcuDataFileName>{customFile}</WasmIcuDataFileName><WasmIncludeFullIcuData>{fullIcu}</WasmIncludeFullIcuData>");
```

Comment:
What does it mean to have fullicu=true, and a custom file from the runtime pack? Would that cause a `icudt.dat`, and the custom file to be included?

Reply:
It should cause only the full ICU to be included: `"$(WasmIcuDataFileName) has no effect when $(WasmIncludeFullIcuData) is set to true.`